README.md

```diff
@@ -41,9 +42,13 @@ This action will open a PagerDuty Maintenance Window for the specified service f
         uses: im-open/open-pagerduty-maintenance-window@v1.0.4
         with:
           pagerduty-api-key: ${{secrets.PAGERDUTY_API_KEY}}
-          service-id: 'P0ABCDE'
           description: 'Code deployment from GitHub Actions'
           minutes: 15
+          service-id: 'P0ABCDE' 
+
+          # Or multiple service IDs
+          service-ids: 'P2W124M,PQQA092,P652LHP,P91AMWC'
```

Comment:
If you're going to do a single example one of these should be commented out.  Or you could include multiple examples in the readme with different ids.